### ISSUE TITLE
Problem with Orr-sommerfeld operator in the documentation

### ISSUE THREAD

**Issue by gokulhari** [chebfun/chebfun#2277]
The example of Orr-Sommerfeld operator in the documentation posted in Chebfun.org does not work as posted. It is producing one positive eigenvalue. Using Chebfun Version 5.7.0 02-Jun-2017.

**Comment by trefethen** [comment]
I am not sure I understand.  Do you mean the example at http://www.chebfun.org/examples/ode-eig/OrrSommerfeld.html?  I just ran this in both R2016a and R2016b, and both worked, giving an
eigenvalue for R=5772.22 of .00005, which is essentially zero.  What problem exactly are you referring to?

**Comment by gokulhari** [comment]
I copy pasted the Orr-Sommerfeld example in the documentation in Chebfun
website:

http://www.chebfun.org/docs/guide/guide07.html

The code is here:

Re = 5772;

B = chebop(-1, 1);

B.op = @(x,u) diff(u,2) - u;

A = chebop(-1, 1);

A.op = @(x,u) (diff(u,4) - 2*diff(u, 2) + u)/Re - ...

    1i*(2*u + (1 - x.^2).*(diff(u, 2) - u));

A.lbc = [0; 0];

A.rbc = [0; 0];

lam = eigs(A, B, 50);

MS = 'markersize';

clf, plot(lam, 'r.', MS, 16), grid on, axis equal

spectral_abscissa = max(real(lam))

In the link above max(real(lam)) is given by -7.82e-5, which I think is
right (is used my own codes, different from Chebfun, to verify). But when I
copy paste the code in Matlab (2018a), I get the following value for
max(real(lam)): spectral_abscissa = 0.0021.  It is close to zero, but it's
not yet supposed become positive.

**Comment by nickhale** [comment]
I'm also running MATLAB 2018a (on Ubuntu on an up-to-date copy of the master branch) and I get 
```
spectral_abscissa =
    -7.835083621178149e-05
```
when using your code above.

**Comment by gokulhari** [comment]
I'm using MacOS High Sierra. I used the most recent version in master branch, commit 602803172f6fcb32c09... . I've attached the output from Matlab's command window using diary
[chebfun_orr_output.txt](https://github.com/chebfun/chebfun/files/2379962/chebfun_orr_output.txt)
.

**Comment by gokulhari** [comment]
You are right, it works fine in Ubuntu. Most likely it's not a bug in Chebfun. Something seems to be going wrong in Mac.

**Comment by abigopal** [comment]
I'm running MacOS High Sierra, but with MATLAB2017b. I get the output:

spectral_abscissa =

    -7.798249627245103e-05


So, from this thread, it looks like the problem is unique to MATLAB2018a
with MacOS, which is strange...

**Comment by gradywright** [comment]
I am running MacOS High Sierra with MATLAB 2018a (9.4.0.813654) and get
```
spectral_abscissa =

   0.002059008040372
```
So this is one more data point on the strange behavior.

**Comment by danfortunato** [comment]
Here are my data points on macOS High Sierra 10.13.6, using the latest chebfun master branch:

MATLAB 2017b (9.3.0.713579)
```
spectral_abscissa =

    -7.620782443996909e-05
```

MATLAB 2018a (9.4.0.813654)
```
spectral_abscissa =

    -7.702195276202216e-05
```

MATLAB 2018b (9.5.0.944444)
```
spectral_abscissa =

    -7.702195276202216e-05
```
